src/components/common/multi-entry/multi-entry.tsx

```diff
@@ -20,16 +23,46 @@ export const MultiEntry = ({
     onAdd,
     children,
     disabledButton,
+    popoverReason,
     subtitle,
 }: IProps) => {
     // =============================================================================
     // RENDER FUNCTIONS
     // =============================================================================
-    return (
-        <MultiEntryAccordionItem title={title} $hasSubtitle={!!subtitle}>
-            {subtitle && <SubtitleText>{subtitle}</SubtitleText>}
-            <>
-                {children}
+
+    const renderPopoverContent = () => {
+        switch (popoverReason) {
+            case EPopoverReason.MAX_ENTRY:
+                return `Limit reached. To add new ${buttonLabel}, remove existing ones first`;
+            case EPopoverReason.EMPTY_OR_INVALID:
+                return `To add new ${buttonLabel}, fill up existing ${buttonLabel} first.`;
+            case EPopoverReason.NO_CONDITION:
+                return `No conditional rendering available.`;
+        }
+    };
```

Comment:
actual popover reason copy should be defined in the corresponding validation / conditional rendering / prefill components

so MultiEntry should accept any copy and render the popover accordingly